### ISSUE TITLE
Merge to tee-dev but with v0.9.26 rollback

### ISSUE THREAD

**Issue by Kailai-Wang** [litentry/litentry-parachain#838]
### Context

As I experimented with dependency update for TEE-worker (as I wanted to test the shielding-key ocall), it seems we have problems right now to update both to v0.9.28:
- use parachain 0.9.28 + tee-worker 0.9.26 won't work, at least not so simple: https://github.com/integritee-network/worker/issues/1000
- update worker to 0.9.28 is blocked by https://github.com/integritee-network/worker/issues/992#issuecomment-1243365213

To not block the dev process, I guess the only way for now is to merge `dev` to `tee-dev` branch, and then rollback the dependency to v0.9.26, so that we have 0.9.26 on both sides.

If you have better ideas, please let me know 

---
:heavy_check_mark: Please set appropriate **labels** and **assignees** if applicable.

**Comment by Kailai-Wang** [comment]
Maybe let's hold on for a while, Integritee mentioned v0.9.28 update should be complete soon

I'd still expect that `retain_mut` problem though, but we can resolve it by patching substrate.

**Comment by Kailai-Wang** [comment]
superseded by https://github.com/litentry/tee-worker/issues/64
